query/src/org/labkey/query/QueryDefinitionImpl.java

```diff
@@ -565,7 +565,7 @@ public TableInfo getTable(@NotNull UserSchema schema, @Nullable List<QueryExcept
                     table = createTable(schema, errors, includeMetadata, null, skipSuggestedColumns);
                 }
 
-                if (null == table || (null != errors && !errors.isEmpty()))
+                if (null == table || (null != errors && !errors.isEmpty() && errors.stream().anyMatch(error -> error instanceof QueryParseException && ((QueryParseException)error).isError())))

```

Comment:
This is subtly different from the similar check here: https://github.com/LabKey/platform/blob/6fb3151a624541e6b2f7133b6527e2415bf191b5/query/src/org/labkey/query/LinkedSchema.java#L345

The other code checks to see if it's explicitly tagged as a warning. This checks if it's an error. If there's some other variant of QueryException in the list, it will continue through.

I haven't tested but I think this will be a more consistent approach:

```suggestion
                if (null == table || (null != errors && !errors.isEmpty() && !errors.stream().allMatch(error -> error instanceof QueryParseException && ((QueryParseException)error).isWarning())))

```

Reply:
Thanks, Josh! That is exactly what i had initially, and it works, then changed it to check for error instead.

Reply:
Thanks, Josh! That is exactly what i had initially, and it works, then changed it to check for error instead.

Reply:
Thanks, Josh! That is exactly what i had initially, and it works, then changed it to check for error instead.